plugins/experimental/buffer_upload/buffer_upload.cc

```diff
@@ -753,6 +753,15 @@ attach_pvc_plugin(TSCont /* contp ATS_UNUSED */, TSEvent event, void *edata)
         memcpy(replacement_host_str, host_hdr_str_val, host_hdr_str_val_len);
         TSDebug(DEBUG_TAG, "Adding host to request url: %s", replacement_host_str);
 
+        const char *colon = strstr(replacement_host_str, ":");
```

Comment:
This is copied from https://github.com/apache/trafficserver/blob/master/plugins/experimental/buffer_upload/buffer_upload.cc#L633
However, I can use strchr instead ^^a